.lighthouse/jenkins-x/release/promote-jx-website.sh

```diff
@@ -8,5 +8,5 @@ git clone https://github.com/jenkins-x/jx-docs.git
 
 cd jx-docs
 sed -i "s/release = \".*\"/release = \"${JX_VERSION}\"/" config.toml
-git commit --allow-empty -a -m "chore: upgrade jx version"
+git commit -a -m "chore: upgrade jx version"
 git push
```

Comment:
The problem as @ankitm123 pointed out is that by not allowing empty commits this command could fail and thus the pipeline would fail. So when not allowing empty commits you should also handle that. For example like this
```suggestion
if git commit -a -m "chore: upgrade jx version"
then
  git push
fi
```